src/scripts/main.js

```diff
@@ -1,3 +1,51 @@
 'use strict';
 
 // write your code here
+const elements = {
+  countriesList: [...document.querySelectorAll('.population')],
+  totalPopulation: document.querySelector('.total-population'),
+  averagePopulation: document.querySelector('.average-population'),
+};
+
+const getNumberFromString = (array) => {
+  const customIndex = 1;
+
+  return array.reduce((total, item, index) => {
+    const currentFactor = Math.pow(1000, array.length - (customIndex + index));
+    const newTotal = total + Number(item) * currentFactor;
+
+    return newTotal;
+  }, 0);
+};
+
```

Comment:
and we also can simplify this one `parseInt(el.textContent.replace(/,/g, ''), 10)`

Reply:
thanks, really shorter :)